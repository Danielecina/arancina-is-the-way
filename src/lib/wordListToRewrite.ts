interface WrongMap {
  wrongWord: string
  correctWord: string
}

const wordListToRewrite : Array<WrongMap> = [
  {
    wrongWord: 'arancino',
    correctWord: 'arancina'
  },
  {
    wrongWord: 'arancini',
    correctWord: 'arancine'
  },
  {
    wrongWord: 'Arancino',
    correctWord: 'Arancine'
  },
  {
    wrongWord: 'Arancini',
    correctWord: 'Arancine'
  }
]

export default wordListToRewrite
