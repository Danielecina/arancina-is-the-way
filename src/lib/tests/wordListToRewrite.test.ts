import wordListToRewrite from '../wordListToRewrite'

describe('wordListToRewrite', () => {
  test('snapshot', () => {
    expect(wordListToRewrite).toMatchSnapshot()
  })
})
