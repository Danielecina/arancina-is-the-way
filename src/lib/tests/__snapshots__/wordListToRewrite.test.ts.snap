// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`wordListToRewrite snapshot 1`] = `
Array [
  Object {
    "correctWord": "arancina",
    "wrongWord": "arancino",
  },
  Object {
    "correctWord": "arancine",
    "wrongWord": "arancini",
  },
  Object {
    "correctWord": "Arancine",
    "wrongWord": "Arancino",
  },
  Object {
    "correctWord": "Arancine",
    "wrongWord": "Arancini",
  },
]
`;
